### ISSUE TITLE
Corrupted lavapipe on Ubuntu 20.04 on Jetson Nano

### ISSUE THREAD

**Issue by Qengineering** [rbonghi/jetson_stats#173]
**Describe the bug**
After `sudo -H pip install -U jetson-stats` on a Jetson Nano with an upgraded Ubuntu 20.04 OS, you get the next warning when booting:

```
Error found when loading /etc/profile:
WARNING: lavapipe is not a conformant vulkan implementation, testing use only.
WARNING: lavapipe is not a conformant vulkan implementation, testing use only.

As a result the session will not be configured correctly.
You should fix the problem as soon as feasible.
```

**To Reproduce**
1. [Upgrade](https://qengineering.eu/install-ubuntu-20.04-on-jetson-nano.html) your Jetson Nano with Ubuntu 20.04 with the Ubuntu upgrade manager.
2. `sudo -H pip install -U jetson-stats`
3. 'sudo reboot'

**Screenshots**
![image](https://user-images.githubusercontent.com/44409029/134178394-6696e39c-07fe-41d1-a12e-060c57c7eaa5.png)

**Additional context**
I know the Jetson Nano comes default with an Ubuntu 18.04 version.
Only a few who need ROS2 will upgrade to Ubuntu 20.04, so not the end of the world here.

**Board**
 - SOC: tegra210
 - ID: 33
 - Board(s): P3449-0000
 - Code Name: porg

**Jetpack**
 - Jetpack: 4.6
 - L4T: 32.6.1

**Jetson-Stats**
 - Version: 3.1.1

**Comment by km4sh** [comment]
Same issue here, wondering any quick fix?

**Comment by SkrilaxCZ** [comment]
@Qengineering I ran into the same issue and deleted directory `/usr/share/vulkan/icd.d` and it went away. Looks like they removed it in Jetpack as well.

**Comment by Qengineering** [comment]
@SkrilaxCZ,

Thank you. It works well. I've updated the procedure on the [website](https://qengineering.eu/install-ubuntu-20.04-on-jetson-nano.html) and added Jtop to the [SD image](https://github.com/Qengineering/Jetson-Nano-Ubuntu-20-image).

**Comment by rbonghi** [comment]
Hi @Qengineering ! Cool post on your blog!

I didn't fix yet this bug but will be soon. I think when will be released as well the new JP 5.0 with the official support to ubuntu 20.04

Meanwhile I pin this workaround :-)

**Comment by Qengineering** [comment]
Another solution: https://github.com/Qengineering/Jetson-Nano-Ubuntu-20-image/issues/5

**Comment by leonpano2006** [comment]
> **Describe the bug** After `sudo -H pip install -U jetson-stats` on a Jetson Nano with an upgraded Ubuntu 20.04 OS, you get the next warning when booting:
> 
> ```
> Error found when loading /etc/profile:
> WARNING: lavapipe is not a conformant vulkan implementation, testing use only.
> WARNING: lavapipe is not a conformant vulkan implementation, testing use only.
> 
> As a result the session will not be configured correctly.
> You should fix the problem as soon as feasible.
> ```
> 
> **To Reproduce** Steps to reproduce the behavior:
> 
> 1. [Upgrade](https://qengineering.eu/install-ubuntu-20.04-on-jetson-nano.html) your Jetson Nano with Ubuntu 20.04 with the Ubuntu upgrade manager.
> 2. `sudo -H pip install -U jetson-stats`
> 3. 'sudo reboot'
> 4. See error
> 
> **Screenshots** ![image](https://user-images.githubusercontent.com/44409029/134178394-6696e39c-07fe-41d1-a12e-060c57c7eaa5.png)
> 
> **Additional context** I know the Jetson Nano comes default with an Ubuntu 18.04 version. Only a few who need ROS2 will upgrade to Ubuntu 20.04, so not the end of the world here.
> 
> **Board**
> 
> * SOC: tegra210
> * ID: 33
> * Board(s): P3449-0000
> * Code Name: porg
> 
> **Jetpack**
> 
> * Jetpack: 4.6
> * L4T: 32.6.1
> 
> **Jetson-Stats**
> 
> * Version: 3.1.1

try to remove mesa-vulkan-drivers
sudo apt remove mesa-vulkan-drivers

**Comment by theofficialgman** [comment]
`mesa-vulkan-drivers` was moved to be a "recoomends" for the apt package libvulkan1 https://packages.ubuntu.com/focal/libvulkan1 in focal and newer distros (not on bionic). As such, any upgrade to focal will install `mesa-vulkan-drivers` which should be removed on the jetsons. this isn't a jtop issue, but an issue with the users upgrade to focal and the qengineering focal image

**Comment by rbonghi** [comment]
I'm testing on a Jetson Xavier with JP5.0.1 DP and I don't see this bug

![image](https://user-images.githubusercontent.com/4551246/173109530-01efa14c-2e55-48cf-a9f1-6e054b7d99d5.png)

**Comment by yhojann-cl** [comment]
Same error on Ubuntu 22.04, using amd radeon rx 6800 xt and native drivers.

**Comment by leonpano2006** [comment]
> Same error on Ubuntu 22.04, using amd radeon rx 6800 xt and native drivers.

On x86_64 host?

**Comment by yhojann-cl** [comment]
> > Same error on Ubuntu 22.04, using amd radeon rx 6800 xt and native drivers.
> 
> On x86_64 host?

Yes:

```bash
$ uname -a
Linux machine 5.15.0-39-generic #42-Ubuntu SMP Thu Jun 9 23:42:32 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by rbonghi** [comment]
This error should now be fixed with the latest release **4.0** try to install and check again
```
sudo -H pip install -U jetson-stats
```
